local-network/deploy/Dockerfile

```
@@ -5,5 +5,8 @@ WORKDIR /usr/src/app
 COPY pyproject.toml ./
 
 RUN pip install --no-cache-dir --editable .
+RUN curl -L https://foundry.paradigm.xyz | bash
+ENV PATH=$PATH:/root/.foundry/bin
+RUN foundryup
```

```suggestion
RUN curl -L https://foundry.paradigm.xyz | bash
ENV PATH=$PATH:/root/.foundry/bin
RUN foundryup
RUN pip install --no-cache-dir --editable .
```
We change `pip` dependencies often.